open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md

```diff
@@ -335,18 +335,16 @@ On the `/predict` route, several things happen when the form is posted:
 
 Using a model this way, with Flask and a pickled model, is relatively straightforward. The hardest thing is to understand what shape the data is that must be sent to the model to get a prediction. That all depends on how the model was trained. This one has three data points to be input in order to get a prediction.
 
-In a professional setting, you can see how good communication is necessary between the folks who train the model and those who consume it in a web or mobile app. In our case, it's only one person, you!
-
 ---
```

Comment:
remove this.

Reply:
This has been removed

Reply:
Could you please double check? Let's make sure all the comments are addressed.